--- mtime_file_watcher.py.orig	2016-09-13 11:10:50.000000000 -0500
+++ mtime_file_watcher.py	2016-09-13 11:07:13.000000000 -0500
@@ -103,6 +103,11 @@
     num_files = 0
     for dirname, dirnames, filenames in os.walk(self._directory,
                                                 followlinks=True):
+      if '/node_modules/' in dirname: continue
+      if '/bower_components/' in dirname: continue
+      if '/.git/' in dirname: continue
+      if '/vendor/nodejs/' in dirname: continue
+      
       if self._quit_event.is_set():
         raise ShutdownError()
       watcher_common.skip_ignored_dirs(dirnames)
